src/test/java/org/broadinstitute/hellbender/cmdline/argumentcollections/IntervalArgumentCollectionTest.java

```diff
@@ -92,6 +93,22 @@ public void testIncludeWithPadding(IntervalArgumentCollection iac){
     }
 
     @Test(dataProvider = "optionalOrNot")
+    public void testIntervalMergingRuleAdjacentMerge(IntervalArgumentCollection iac){
+        iac.addToIntervalStrings("1:1-100");
+        iac.addToIntervalStrings("1:101-200");
+        iac.intervalMerging = IntervalMergingRule.ALL;
+        Assert.assertEquals(iac.getIntervals(hg19GenomeLocParser.getSequenceDictionary()), Arrays.asList(new SimpleInterval("1", 1, 200)));
+    }
+
+    @Test(dataProvider = "optionalOrNot")
+    public void testIntervalMergingRuleAdjacentNoMerge(IntervalArgumentCollection iac){
+        iac.addToIntervalStrings("1:1-100");
+        iac.addToIntervalStrings("1:101-200");
+        iac.intervalMerging = IntervalMergingRule.OVERLAPPING_ONLY;
+        Assert.assertEquals(iac.getIntervals(hg19GenomeLocParser.getSequenceDictionary()).size(), 2);
+    }
+
```

Comment:
Add a test case that uses the INTERSECTION set rule in combination with both of the merging rules